CODIGO/General.bas

```diff
@@ -1134,7 +1134,7 @@ On Error GoTo Main_Err
     UpdateLights = False
     LastOffset2X = 0
     LastOffset2Y = 0
-    Call SwitchMap(UserMap)
+    Call SwitchMap(UserMap, UserMap)
```

Comment:
This is not clear, switch map should only do one thing. I don't think it's a good idea to add a new argument and also calling it like this `Call SwitchMap(UserMap, UserMap)` does not make much sense.

We need to think carefully a way to implement this without making the code too difficult to understand.

Reply:
`Call SwitchMap(UserMap, UserMap)` it does not make much sense to pass the same argument twice, lets make changes to the function to make it optional